lib/cache.js

```diff
@@ -2,6 +2,8 @@
 
 // Impress Application Full Memory Cache
 
+const DEFAULT_WATCH_INTERVAL = 2000;
```

Comment:
I think it will create a lot of unnecessary fs calls and cpu load at 2 seconds. I think we should maybe use 2000 for `test` mode but have it at maybe 1 minute in `prod`, wdyt?

Reply:
This doesn't means that we will create multiple timers or check fs every 2 seconds, this just mean that updated files will not be recached immediately after receiving watch event and we will wait at least 2 seconds (by default) for more watch events to process them together. Server configuration may override this interval. For dev mode it's better to minimize this interval, for production it may be increased but not 60 sec, maybe to 10-15 sec. @lundibundi @nechaido

Reply:
Could we then rename it to `DEFAULT_WATCH_DEBOUNCE_TIME` as that will make it a bit clearer IMO.
I'm +1 to having it 2000 for dev mode and 10000 in production.

Could we use something like
```js
const DEFAULT_WATCH_DEBOUNCE_TIME = process.env.IMPRESS_MODE === 'prod' ? 10000 : 2000;
```
?